### PR TITLE
UpdateBudget.h

### PR DESCRIPTION
**Pull request by AhmedHamdyKotp**

### CODE REVIEW

File: include/Budget.h [hunk 1]
```diff
@@ -1,27 +1,30 @@
...
-#include <vector>
 #include <map>
 
-using namespace std;
-
 
-class Budget
-{
+using namespace std;
+class Budget :  Person {
```
Comment (OsamaAshraf01) [inline include/Budget.h]: Budget Class doesn't inherit from Person Class

File: include/Budget.h [hunk 2]
```diff
@@ -1,27 +1,30 @@
...
+    double total_budget;
 public :
+    //Constructor
     Budget();
-    double total_budget;
-	void add_category(const string& category_name, double amount);
-    double getBalance(const string& category_name) const;
-    void spend(const string& category_name, double amount);
+
+    //setters
```
Comment (OsamaAshraf01) [inline include/Budget.h:17]: those three functions are methods not setters. Setters usually start with 'set'. In this example, setTotalBudget only

Comment (OsamaAshraf01) [reply include/Budget.h:17]: don't delete those functions, just move them to methods section.

**Review by OsamaAshraf01 — CHANGES_REQUESTED** [review]

**Comment by AhmedHamdyKotp** [comment]
Done

**Pull request merged**